bin/yaml/solidity.yaml

```diff
@@ -11,3 +11,11 @@ compilers:
       - 0.6.12
       - 0.7.6
       - 0.8.13
+  solidity_zkevm:
+    type: solidity-zkevm
```

Comment:
Solidity has `type: solidity` because it uses a custom installable (`SolidityInstallable` in `bin/lib/installable/solidity.py`. I don't see anything defined in this PR for this new type, so I guess this was a copy/paste error?

Reply:
Ah, now I see that the fields in yaml are accessible from the python implementation.
Let's probably continue [this discussion](https://github.com/compiler-explorer/compiler-explorer/pull/4398#issuecomment-1345573332) here.

Reply:
I'm going to look into the `poetry` issue - that's really odd as I just tried it on a clean install and didn't see this issue but I'll dig in a little. Sorry it didn't work :(

Reply:
@hedgar2017 if you still have the `infra` repro around can you see if you have a `poetry.toml` file that's empty in your infra directory. I suspect somehow it got truncated and so it didn't install to the "right" VM.

Reply:
Did this fix it for you @hedgar2017 ?

Reply:
Thanks a lot for your fix!
I couldn't try yet, but I will after the holidays for sure!

Reply:
Hey @mattgodbolt, just to clarify, did you just want to me to show the `poetry.toml` contents?
Here they are:
```
# automatically generated by Makefile\nvirtualenvs.path = "/home/hedgar/src/compiler-explorer-infra/.poetry/virtualenvs"
```
The file indeed contains just a single comment line.
Getting the same error as before.

Reply:
Very odd! Your poetry.toml seems very out of date. The fix from the end of December should have replaced that auto-generated poetry with the one checked into source control. Thanks for posting the poetry.toml; I at least know what went wrong in the old way of doing things.

Can you double-check you merged in main into your branch and give it another go?

Reply:
Oh strange, Github doesn't let me update the branch.
I'll try manually soon!

Reply:
All is working now. I believe we could resolve this thread.